frontend/src/routes/Governance/components/GovernanceEmptyState.tsx

```diff
@@ -86,7 +86,7 @@ export function GovernanceCreatePolicysetEmptyState() {
             action={
                 <div>
                     <AcmButton component={Link} variant="primary" to={NavigationPath.policies}>
-                        {'Create policy set'}
+                        {t('Create policy set')}
```

Comment:
Why is it "Create **a** policy" but just "Create policy set"? @dockerymick Should the buttons have "a" in them or not?

Reply:
Actually the 'a' is not in design so updating that now